src/serac/numerics/functional/boundary_integral.hpp

```diff
@@ -51,11 +53,14 @@ class BoundaryIntegral {
    * @note The @p Dimension parameters are used to assist in the deduction of the dim template parameter
    */
   template <int dim, typename lambda_type, typename qpt_data_type = void>
-  BoundaryIntegral(int num_elements, const mfem::Vector& J, const mfem::Vector& X, const mfem::Vector& normals,
-                   Dimension<dim>, lambda_type&& qf)
+  // BoundaryIntegral(int num_elements, const mfem::Vector& J, const mfem::Vector& X, const mfem::Vector& normals,
+  //                 Dimension<dim>, lambda_type&& qf)
+  BoundaryIntegral(int, const mfem::Vector& J, const mfem::Vector& X, const mfem::Vector& normals, Dimension<dim>,
+                   lambda_type&&)
       : J_(J), X_(X), normals_(normals)
 
   {
+#if 0
```

Comment:
Can you add a comment why this is ifdef'd out or remove it?

Reply:
it's just a temporary `#ifdef` to get some of the tests compiling before the variadic support is implemented for the boundary integral kernels. This code should be updated and re-enabled before this PR is merged in.